client/src/components/Navbars/TopNavbar.tsx

```diff
@@ -213,9 +211,9 @@ const TopNavbar: FC<Props> = ({ position = 'fixed', enableOnScrollEffect = true
             </Button>
           </Box>
           <Box sx={{ flexGrow: 1 }}>
```

Comment:
Is there any visual difference including and excluding this box? It seems like it just adds an extra unnecessary element to the DOM tree.